README.md

```diff
@@ -234,6 +234,34 @@ For other errors, please [open an issue](https://github.com/BlueBrain/CoreNeuron
 
 ## Developer Build
 
+#### Running Unit and Integration Tests
+
+As **CoreNEURON** is mostly used as a compute library of **NEURON** it needs to be incorporated with **NEURON** to test most of its functionality. Consequently its tests are included in the NEURON repository. To enable and run all the tests of **CoreNEURON** you need to add the `-DNRN_ENABLE_TESTS=ON` CMake flag in **NEURON**.
+Those tests include:
+* Unit tests of NEURON
+* Unit tests of CoreNEURON
+* Integration tests of NEURON
+* Integration tests of CoreNEURON
+* Integration tests between NEURON and CoreNEURON
+* [ringtest](https://github.com/neuronsimulator/ringtest) tests with NEURON and CoreNEURON
+* [testcorenrn](https://github.com/neuronsimulator/testcorenrn) tests with NEURON and CoreNEURON
+Some of those tests are going to be also run with various backends in case that those are enabled (for example with GPUs).
+To build NEURON with CoreNEURON and run the tests you need to:
+
+```bash
+cd nrn/build
+cmake .. \
```

Comment:
Personally I would add `-G Ninja` 😇

Reply:
Haha I think that since this is not mandatory we can keep it aside :stuck_out_tongue_closed_eyes:

Reply:
yeah, I would keep this aside (ninja is not always installed on end-users machine, right?)